### ISSUE TITLE
awssdkv1 is missing in 0.2.15 release

### ISSUE THREAD

**Issue by askerkhakh** [localstack/localstack-java-utils#70]
From my understanding, it was introduced in https://github.com/localstack/localstack-java-utils/commit/7e8c8228e85fada76554bc7473679e4da76ab472. Not sure what was the idea, but I guess maven compiler plugin excludes couldn't exclude awssdkv1 and awssdkv2 simultaneously :slightly_smiling_face:

**Comment by whummer** [comment]
Good catch, thanks @askerkhakh . Do you think you could create a small pull request to fix this? We can also look into it over the next couple of days, but there may be some delay due to a few other critical issues, unfortunately.. Thanks for your help!

**Comment by askerkhakh** [comment]
I'm a bit out of the loop, what was the original issue and what was the goal with these exclusions?

**Comment by lakkeger** [comment]
Hi! We just wanted to follow up to see whether your issue has been resolved. Were you able to get it working with the latest version of LocalStack? We would appreciate your feedback!
